### DISCUSSION TITLE
SumutraPDF 3.6.16779 do not show embedded files

### DISCUSSION THREAD

**Discussion by lywind** [sumatrapdfreader/sumatrapdf#4560] [General]
In SumutraPDF 3.5.2, I can view and save embedded files in PDF
![image](https://github.com/user-attachments/assets/0c8d34cb-85c9-447c-ae49-90c370c57356)
However, in version 3.6 pre-release, the embedded files are disappeared.
![image](https://github.com/user-attachments/assets/ec1d1bf9-22b9-4bc7-87eb-76add72bdf24)

**Comment by GitHubRulesOK** [comment]
There are several different types of embedding and thus a sample of a type that is not working is needed to see why, even if its your own made up similar file. But as always the one you noticed is best.

Thus raise an issue with your problem sample. Without samples there is nothing unseen to fix.

**Reply by lywind** [reply]
Hello, the attached PDF file is the file which can reproduce the problem. In fact, it seems that all PDFs with attachments are not shown in version 3.6.16779.
[S32K-RM.pdf](https://github.com/user-attachments/files/17165904/S32K-RM.pdf)

**Reply by GitHubRulesOK** [reply]
Hmm MuPDF does not naturally have an attachments window and 3.5.2 was showing them so is  a regression in SumatraPDF 
There are still quite a few related from some time ago but on review none seem to mention the most recent change.


![image](https://github.com/user-attachments/assets/7a4f8be0-5f24-4476-88b5-82676b5dda32)

**Reply by GitHubRulesOK** [reply]
I have migrated this discussion to open issue https://github.com/sumatrapdfreader/sumatrapdf/issues/4563

**Comment by musikai** [comment]
I just realized that embedded files (in my PDF as well as in the example PDF from here) don't show up in 3.6.16779 but they do in 3.6.15987.
![grafik](https://github.com/user-attachments/assets/55be8128-9743-4300-9e70-b2924e89c94c)

**Comment by gawkla** [comment]
The PDF created and sent by the Polish bank mBank contains embedded files (statements). Some statements are embedded as HTML files. Is it possible to open them directly in the browser without saving them? Currently, I can see the embedded files, but when I click on them, nothing happens.

![obraz](https://github.com/user-attachments/assets/3cd2bb41-afe2-4ade-812e-a57b8dd44eba)
